cpp/src/arrow/filesystem/s3_test_util.cc

```diff
@@ -91,9 +119,22 @@ Status MinioTestServer::Start() {
   impl_->connect_string_ = GenerateConnectString();
   ARROW_RETURN_NOT_OK(impl_->server_process_->SetExecutable(kMinioExecutableName));
   // NOTE: --quiet makes startup faster by suppressing remote version check
-  impl_->server_process_->SetArgs({"server", "--quiet", "--compat", "--address",
-                                   impl_->connect_string_,
-                                   impl_->temp_dir_->path().ToString()});
+  std::vector<std::string> start_args = {"server", "--quiet", "--compat", "--address",
+                                         impl_->connect_string_};
+  if (enable_tls) {
```

Comment:
We can always enable TLS.

Reply:
ok, removed the enable_tls flag and always enable the TLS